i18n/.js/src/test/scala/io/udash/i18n/LocalTranslationProviderTest.scala

```diff
@@ -9,7 +9,7 @@ class LocalTranslationProviderTest extends UdashFrontendTest {
 
   "LocalTranslationProvider" should {
     "provide translations without argument" in {
-      val translator = new LocalTranslationProvider(
+      implicit val translator = new LocalTranslationProvider(
```

Comment:
Should we migrate `RemoteTranslationProviderTest` similarly?

Reply:
It's not using `getTranslatedString` helper so we fortunately don't need to.